src/CodeStyle/Core/CodeFixes/xlf/CodeStyleFixesResources.zh-Hans.xlf

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hans" original="../CodeStyleFixesResources.resx">
```

Comment:
Is this a necessary change? or does it matter?

Reply:
@JoeRobich  we will update PR within an hour

Reply:
@JoeRobich PR is updated with correct language code